mpt/impl/smtp/core/src/main/java/org/apache/james/mpt/smtp/ForwardSmtpTest.java

```diff
@@ -66,6 +66,7 @@ public ForwardSmtpTest() throws Exception {
     }
 
     @Before
+    @SuppressWarnings("deprecation")
```

Comment:
Yes, deprecation is on getIpAddress method.
We should modify DNSServier.getByName method in order to return `InetSocketAddress` instead of `InetAddress`, that way we will be able to bind to localhost:mappedPort.